.unreleased/LLT-5402

```diff
@@ -0,0 +1 @@
+Use batcher for stun, vpn and proxied peers persistent keepalives
```

Comment:
As this PR is requiring additional QA cycle - lets wait for its merge to happen after v5.0.0 release